src/pages/amazon-sales-channel/best-practices/logging-troubleshooting.md

```diff
@@ -0,0 +1,253 @@
+    ```text
+    ›   Error: PUT http://localhost:3233/api/v1/namespaces/guest/actions/amazon-app/scheduled-amazon-sync-products Returned HTTP 400 (Bad Request) --> "The request content was malformed:
+    ›   requirement failed: duration 400000 milliseconds exceeds allowed threshold of 300000 milliseconds"
+    ```
+
+  * You have a dependency on the `lib-state` storage library, which causes the following error:
+
+    ```text
+    "code": "ERROR_BAD_ARGUMENT",
+    "message": "[TvmLib:ERROR_BAD_ARGUMENT] ow.auth" is required"
+    "stacktrace": "TvmLibError: [TvmLib:ERROR_BAD_ARGUMENT] "ow.auth" is required
+    ```
+
+  * You have triggers (scheduled runtime actions) configured. This will show the following error:
+
+    ```text
+    Error: POST http://localhost:3233/api/v1/namespaces/whisk.system/actions/alarms/interval Returned HTTP 403 (Forbidden) --> "The supplied authentication is not authorized to access 'whisk.system/alarms'."
+    ```
+
+### Timeout issues
+
+Based on the development team's experience developing Amazon Sales Channel app, a 503 error may occur when an action exceeds its allotted time. However, this error message may not always provide sufficient information, especially when the action is invoked from a React UI. In such cases, it is recommended to investigate further using the activation ID to gain more insights. If the result obtained from activation ID indicates a timeout error, it will typically display a message such as:
+
+```json
+{
+  "error": "The action exceeded its time limits of 60000 milliseconds."
+}
+```
+
+In such instances, investigate the root cause of the timeout error, which could potentially be due to a slow response from an external API. If so, you might consider reviewing the specific API call that is taking excessive time. Additionally, adjusting the timeout duration for your action may be a solution. Note that the maximum allowable timeout for a blocking action is set at 60 seconds.
+
+## Additional tips
+
+* When a new log forwarding configuration is set to an existing destination, it overwrites the previous configuration but does not remove any logs previously sent there. When the destination of the logs is changed (for example from Splunk back to Adobe I/O Runtime), logs previously sent to Splunk are not removed. And when setting forwarding back to Splunk once again, the previous logs are not reset or removed.
+
+* Once you start forwarding logs to Splunk HEC, running the `aio rt logs` command will return the following warning:
+
+  `Namespace is configured with custom log forwarding destination: 'splunk_hec'. Please use corresponding logging platform to view logs`
+
+* If you receive the `Too many debug requests in the last minute` error, consider disabling the `'X-OW-EXTRA-LOGGING': 'on'` header in your runtime action. This setting activates throttling to your App Builder application, and some requests will be denied. Use log-forwarding to reduce the number of messages being logged instead.
```

Comment:
Can we add an additional section for more resources? 

Join the #app-builder-community slack channel in the Commerce Engineering [Slack workspace](magentocommeng.slack.com) to ask questions directly to teams and partners working with App Builder.

Check [Common Troubleshooting](https://developer.adobe.com/app-builder/docs/getting_started/common_troubleshooting/) documentation for the latest updates.

Reply:
Added the links to the overview topic